api/src/routes/api/mock-sas9.ts

```diff
@@ -0,0 +1,145 @@
+import { readFile } from '@sasjs/utils'
+import express from 'express'
+import path from 'path'
+
+const mockSas9Router = express.Router()
+
+const { MOCK_SERVERTYPE } = process.env
+
+let loggedIn: boolean = false
+
+mockSas9Router.get('/SASStoredProcess', async (req, res) => {
+  if (!loggedIn) {
+    res.redirect('/SASLogon/login')
+    return
+  }
+
+  const filePath = path.join(
+    process.cwd(),
+    'mocks',
+    'generic',
+    'sas9',
+    'sas-stored-process'
+  )
+
+  let file
+
+  try {
```

Comment:
I think we should use `then-catch` block instead of `try-catch`.

Reply:
If you like to use try-catch block to make it consistent with other routes as I saw we are using these in other routes too, then you can respond with a success response in the try block. No need to add another try block at the end